src/form/Select/SelectInput/SelectInputTheme.ts

```diff
@@ -66,7 +66,7 @@ const baseTheme: SelectInputTheme = {
     inputContainer: 'flex-wrap'
   },
   chip: {
-    base: 'cursor-pointer flex text-xs leading-none box-border mr-1 my-px px-1 py-1 rounded border-solid border-transparent',
+    base: 'cursor-pointer flex text-base leading-none box-border mr-1 my-px px-1 py-1 rounded border-solid border-transparent',
```

Comment:
Can u test the multi-select w/ chips to ensure this is correct and post a screenshot?

Reply:
yea here's multi-select 
![image](https://github.com/reaviz/reablocks/assets/40581813/033a94ba-9ec8-424e-8138-df87d6491c2e)
![image](https://github.com/reaviz/reablocks/assets/40581813/af5cfe70-dcf7-4e09-b3b5-e7ecaba8d7d9)
![image](https://github.com/reaviz/reablocks/assets/40581813/163847fa-8457-4413-86dd-8db1a39bed0d)

that's size 14px for the placeholder and chips, the dropdown contents 12px

it actually didn't change anything from the current styling in this storybook cuz this repo doesn't have `text-xs` so that wasn't doing anything and font size defaults to 14px which is same as `text-base`

This was only having an effect on repos where `text-xs` exists

Reply:
updated chip font size to be smaller so it doesn't increase the input height

https://github.com/reaviz/reablocks/assets/40581813/93d17e45-af37-4b3b-b64e-cd288916450f